base/sysinfo.jl

```diff
@@ -200,7 +212,19 @@ function _cpu_summary(io::IO, cpu::AbstractVector{CPUinfo}, i, j)
     end
     println(io)
 end
+"""
+    Sys.cpu_summary()
+
+Prints a summary of CPU information, organizing and displaying aggregated data for CPUs with the same model.
```

Comment:
The style guide says that documentation should be imperative.
```suggestion
Print a summary of CPU information, organizing and displaying aggregated data for CPUs with the same model.
```